packages/block-editor/src/components/media-placeholder/README.md

```diff
@@ -85,7 +85,15 @@ If false the default placeholder style is used.
 - Type: `Boolean`
 - Required: No
 - Default: `false`
-- Platform: Web
+- Platform: Web | Mobile
+
+### disableMediaSelection
+
+Whether render a dropzone/placholder without any other additional UI.
```

Comment:
I think this flag if enabled will render a drop zone but not a placeholder we can update the text to: "Whether render a dropzone without any other additional UI".